pkg/asset/rhcos/image.go

```diff
@@ -67,7 +67,6 @@ func (i *Image) Generate(p asset.Parents) error {
 		//TODO(serbrech): change to right image once available.
 		osimage = "/resourceGroups/rhcos_images/providers/Microsoft.Compute/images/rhcostestimage"
 	case baremetal.Name:
-		// FIXME: baremetal
 		osimage, err = rhcos.QEMU(ctx)
```

Comment:
This doesn't need fixing:

```
using the same image as libvirt is correct for the bootstrap VM
we do have: https://github.com/openshift-metal3/kni-installer/issues/37
which covers "use the right image for masters and workers"
```

Reply:
Yeah also see https://github.com/openshift-metal3/kni-installer/pull/100 where we need to consume two different images which fixes that issue, so I bypassed this abstraction - I think removing this FIXME is reasonable.

Reply:
This is ... ugh ...

Some of the underlying assumptions of the `rhcos.Image` abstraction:

1. The same image is used for bootstrap, masters, and workers
2. That image information is included in the generated machines manifests and also to terraform

I think we should follow that basic idea and just have an "oh hey, our bootstrap node boot image is different!" special case

Reply:
Yeah agree we need a better abstraction here which caters for the situation where the bootstrap and master images are different, I hacked around it in #100 but would welcome feedback on possible approaches there.